src/portal/static/portal/js/index.js

```diff
@@ -0,0 +1,12 @@
+$('#menu-toggle').click(function(event) {
+    event.preventDefault();
+    $('#wrapper').toggleClass('toggled');
+});
+
+$(window).scroll(function() {
+    if ($(document).scrollTop() > 50) {
+        $('#top-nav').addClass('fade');
+    } else {
+        $('#top-nav').removeClass('fade');
+    }
+});
```

Comment:
I was just looking at this code, and I thought that it would be better if you assign a `$('#top-nav')` to a variable, that way jQuery won't be searching all DOM for this element each time user is scrolling the website, e.g. you could do something like this:
```
var topNav = $('#top-nav');
var document = $(document);
$(window).scroll(function() {
     if (document.scrollTop() > 50) {
         topNav.addClass('fade');
     } else {
         topNav.removeClass('fade');
     }
 }); 
```

I'm not in this project so it's up to you if you want to change it :)